Initial v1 docs (#15)
buape/kiai-docs: pages/commands/ping.mdx

# Ping Command

## Syntax

```
/ping
```

## Permissions

This command is available to everyone.

## Description

> The `/ping` command displays the current connection speed of the bot to Discord.

Reviewer: ```suggestion
The `/ping` command displays the latency between the bot and Discord.
```
